Detectors/CTP/macro/GetScalersForRun.C

```diff
@@ -10,9 +10,11 @@
 // or submit itself to any jurisdiction.
 
 #if !defined(__CLING__) || defined(__ROOTCLING__)
+#include <TMath.h>
 #include <CCDB/BasicCCDBManager.h>
 #include <DataFormatsCTP/Configuration.h>
-#include "ctpRateFetcher.h"
+#include <DataFormatsParameters/GRPLHCIFData.h>
+#include "Common/CCDB/ctpRateFetcher.h"
```

Comment:
We should guard this and the places where we use `ctpRateFetcher` with a #ifdef or we should move the full macro to O2Physics